wunderio/drupal-project-k8s
# Feature/slt 318

Restores database and files from a backup, given a date.
Refactored post-release-command in _helpers.tpl to separate drupal installation + followup deployments from backup deployments.
Introduces postRestoreCommand variable in values.yaml to specify a command to execute after backup is imported.

Eventual merge is dependant on https://github.com/wunderio/silta-circleci/pull/100 being merged first.

Testing:
1. Create an environment, and backup
2. Touch some files in public-files, add articles in drupal.
3. Change CircleCI config.yml to use `backup` branch -> `silta/silta@dev:backup`
3. Push a commit with message #DestroyCurrentAndRestoreBackup=[yyyy-mm-dd]-[branch], date of the backup made.
4. Check that another backup is made - to save the current state, before the targeted backup is imported.
5. Extra files from step 2 are removed. Drupal articles should be gone.
